Sources/MapboxDirections/Waypoint.swift

```diff
@@ -215,7 +215,13 @@ extension Waypoint: Equatable {
 
 extension Waypoint: CustomStringConvertible {
     public var description: String {
-        return name ?? "<latitude: \(coordinate.latitude); longitude: \(coordinate.longitude)>"
+        return Mirror(reflecting: self).children.compactMap({
+            if let label = $0.label {
+                return "\(label): \($0.value)"
+            }
+            
+            return ""
+        }).joined(separator: "\n")
```

Comment:
Not bad. What do you think about further customizing the Quick Look implementation of this type to show the target coordinate when present instead of the main coordinate?

Reply:
Interestingly `debugQuickLookObject` doesn't work for me in Swift, when using Objective-C it works as expected. Will add `targetCoordinate ?? coordinate` though.